plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -1032,6 +1028,17 @@ private boolean isMaterializedView(Table table)
                 && table.getParameters().containsKey(STORAGE_TABLE);
     }
 
+    @Override
+    public List<SchemaTableName> listMaterializedViews(ConnectorSession session, Optional<String> schemaName)
```

Comment:
Let's add test in TestIcebergMetadataListing.java..

Should https://github.com/trinodb/trino/pull/8153 land first?

Reply:
yes, let's land that one first